### PR TITLE
Couger01/add units

### PR DESCRIPTION
**Pull request by couger01**

### CODE REVIEW

**Comment by rbovill** [comment]
All tests pass on my end.  Changes look good to merge.

File: sal_interfaces/CBP/CBP_Commands.xml [hunk 1]
```diff
@@ -14,9 +14,9 @@
         <Explanation>http://sal.lsst.org</Explanation>
         <item>
             <EFDB_Name>clearFault</EFDB_Name>
-            <Description></Description>
+            <Description>Needed by SAL</Description>
```
Comment (tribeiro) [inline sal_interfaces/CBP/CBP_Commands.xml]: Standard is:

*Ignored Attributes in Topics*

Many generic commands have an ignored attribute. This is due to a requirement from the API to not have empty topics. If you are adding a command to your CSC that does not require an attribute, it still must contain a "dummy" one. In order to maintain consistency across this use case, the attribute must be called value, be of type boolean, be given the following description: "Attribute required by the SAL API, but is unused." and have the following units: n/a (pending verification).

https://confluence.lsstcorp.org/display/SYSENG/SAL+constraints+and+recommendations

File: sal_interfaces/CBP/CBP_Commands.xml [hunk 2]
```diff
@@ -35,7 +35,7 @@
             <EFDB_Name>azimuth</EFDB_Name>
             <Description>The amount to move the azimuth by.</Description>
```
Comment (tribeiro) [inline sal_interfaces/CBP/CBP_Commands.xml]: Is this an offset or a position? It is not clear from the description.

File: sal_interfaces/CBP/CBP_Commands.xml [hunk 3]
```diff
@@ -54,7 +54,7 @@
             <EFDB_Name>altitude</EFDB_Name>
             <Description>The amount to move the altitude by.</Description>
```
Comment (tribeiro) [inline sal_interfaces/CBP/CBP_Commands.xml]: Is this an offset or a position? It is not clear from the description.

File: sal_interfaces/CBP/CBP_Telemetry.xml [hunk 4]
```diff
@@ -10,42 +10,42 @@
             <EFDB_Name>panic</EFDB_Name>
             <Description>The status of the CBP.</Description>
             <IDL_Type>float</IDL_Type>
-            <Units></Units>
+            <Units>unitless</Units>
             <Count>1</Count>
         </item>
         <item>
             <EFDB_Name>azimuth</EFDB_Name>
             <Description>The status of the azimuth</Description>
```
Comment (tribeiro) [inline sal_interfaces/CBP/CBP_Telemetry.xml]: "The status of the azimuth **axis**" or **motor**

File: sal_interfaces/LinearStage/LinearStage_Commands.xml [hunk 5]
```diff
@@ -66,16 +66,16 @@
 		<Explanation>http://sal.lsst.org</Explanation>
 		<item>
 			<EFDB_Name>homed</EFDB_Name>
```
Comment (tribeiro) [inline sal_interfaces/LinearStage/LinearStage_Commands.xml]: called `value`, be of type boolean, be given the following description: "Attribute required by the SAL API, but is unused."

File: sal_interfaces/LinearStage/LinearStage_Commands.xml [hunk 6]
```diff
@@ -92,16 +92,16 @@
 		<Explanation>http://sal.lsst.org</Explanation>
 		<item>
 			<EFDB_Name>stopped</EFDB_Name>
```
Comment (tribeiro) [inline sal_interfaces/LinearStage/LinearStage_Commands.xml]: called value, be of type boolean, be given the following description: "Attribute required by the SAL API, but is unused."

File: sal_interfaces/LinearStage/LinearStage_Events.xml [hunk 7]
```diff
@@ -28,7 +28,7 @@
             <EFDB_Name>heartbeat</EFDB_Name>
             <Description>Beats the drum.</Description>
```
Comment (tribeiro) [inline sal_interfaces/LinearStage/LinearStage_Events.xml]: ?

File: sal_interfaces/TunableLaser/TunableLaser_Commands.xml [hunk 8]
```diff
@@ -14,9 +14,9 @@
         <Explanation>http://sal.lsst.org</Explanation>
         <item>
             <EFDB_Name>startPropagateLaser</EFDB_Name>
```
Comment (tribeiro) [inline sal_interfaces/TunableLaser/TunableLaser_Commands.xml]: called value, be of type boolean, be given the following description: "Attribute required by the SAL API, but is unused."

File: sal_interfaces/TunableLaser/TunableLaser_Commands.xml [hunk 9]
```diff
@@ -33,9 +33,9 @@
         <Explanation>http://sal.lsst.org</Explanation>
         <item>
             <EFDB_Name>stopPropagateLaser</EFDB_Name>
```
Comment (tribeiro) [inline sal_interfaces/TunableLaser/TunableLaser_Commands.xml]: called value, be of type boolean, be given the following description: "Attribute required by the SAL API, but is unused."

File: sal_interfaces/TunableLaser/TunableLaser_Commands.xml [hunk 10]
```diff
@@ -52,9 +52,9 @@
         <Explanation>http://sal.lsst.org</Explanation>
         <item>
             <EFDB_Name>clearFaultState</EFDB_Name>
```
Comment (tribeiro) [inline sal_interfaces/TunableLaser/TunableLaser_Commands.xml]: called value, be of type boolean, be given the following description: "Attribute required by the SAL API, but is unused."

File: sal_interfaces/TunableLaser/TunableLaser_Events.xml [hunk 11]
```diff
@@ -43,7 +43,7 @@
             <EFDB_Name>heartbeat</EFDB_Name>
             <Description>Beats the drum.</Description>
```
Comment (tribeiro) [inline sal_interfaces/TunableLaser/TunableLaser_Events.xml]: ?

File: sal_interfaces/CBP/CBP_Commands.xml (hunk 1 above)
Comment (couger01) [reply sal_interfaces/CBP/CBP_Commands.xml]: https://ts-xml.lsst.io/sal_constraints_and_recommendations.html#ignored-attributes-in-topics

File: sal_interfaces/LinearStage/LinearStage_Events.xml (hunk 7 above)
Comment (couger01) [reply sal_interfaces/LinearStage/LinearStage_Events.xml]: Copy and paste from Russell's heartbeat event.

File: sal_interfaces/TunableLaser/TunableLaser_Events.xml (hunk 11 above)
Comment (couger01) [reply sal_interfaces/TunableLaser/TunableLaser_Events.xml]: Copy and paste from Russell's heartbeat event.

**Pull request merged**